tests/integration/datadict/submission/utils.py

```diff
@@ -22,6 +23,20 @@
     "treatment.json",
 ]
 
+extended_data_fnames = [
```

Comment:
could you replace this by `extended_data_fnames = data_fnames + ["experimental_metadata.json"]` and add a comment about why you don't just add it to `data_fnames`?